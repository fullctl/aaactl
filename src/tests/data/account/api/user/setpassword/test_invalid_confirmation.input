{
  "password": "test",
  "password_new": "updatedpassword",
  "password_confirmation": "passwordupdated"
}
