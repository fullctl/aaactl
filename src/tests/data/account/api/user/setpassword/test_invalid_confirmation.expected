{"data": [], "errors": {"password_confirmation": ["Password confirmation needs to match password"]}, "profiling": {"queries": 0}}
